src/main.js

```diff
@@ -233,6 +233,13 @@ define('peaks', [
      * @type {Function}
      */
     this.logger = console.error.bind(console);
+
+    /**
+     * Destroy anything that needs to be destoryed
+     */
+    this.destroy = function() {
+      this.waveform.destroy();
```

Comment:
`this.waveform` can be undefined when bad shit happens.

I might add an assertion rather than a check:

```js
if (typeof this.waveform !== 'object') {
   throw new Error('attempted to destroy a peaks.js instance without the waveform being created');
}
```